aws/cloudfront_distribution_configuration_structure_test.go

```diff
@@ -585,9 +574,6 @@ func TestCloudFrontStructure_flattenOrigin(t *testing.T) {
 	if out["origin_path"] != "/" {
 		t.Fatalf("Expected out[origin_path] to be /, got %v", out["origin_path"])
 	}
-	if !out["custom_origin_config"].(*schema.Set).Equal(in["custom_origin_config"].(*schema.Set)) {
```

Comment:
`custom_origin_config` itself contains the `origin_ssl_protocols` set, which are not stable to test in this manner.